src/components/unique-token/UniqueTokenAttributes.tsx

```diff
@@ -47,18 +21,49 @@ const UniqueTokenAttributes = ({
   slug,
   traits,
 }: UniqueTokenAttributesProps) => {
-  const sortedTraits = (sortList(
-    traits,
-    'trait_type',
-    'asc'
-  ) as typeof traits).map(trait => ({
-    ...trait,
-    color,
-    disableMenu,
-    slug,
-  }));
+  const sortedTraits = useMemo(
+    () =>
+      (sortList(traits, 'trait_type', 'asc') as typeof traits)
+        .filter(trait => !isHttpUrl(trait.value))
+        .filter(trait => trait.value && trait.trait_type)
+        .sort(uniqueAssetTraitDisplayTypeCompareFunction)
+        .map(transformUniqueAssetTraitsForPresentation)
+        .map(trait => ({
+          ...trait,
+          color,
+          disableInteraction: disableMenu || trait.disableMenu,
+          slug,
+        })),
```

Comment:
I don't really like this part. Could you please verify it's not slow? Also, that we won't rerun this thing unnecessary? I imagine such code being really slow on low-end androids. I'm sure we can reduce this to 3 loops by combining both `sort`, both `filter`, and `map`.